joomla/joomla-cms
# [4.0] Codemirror proper Custom Element

Pull Request for Issue # .

### Summary of Changes
- Since we are already concatenating a bunch of files for the code mirror I've concatenated 2 more...
- The custom element is not using Async/Await as this will fail to anything older than current ver of Evergreen browsers
- The element is using correctly mutation observers
- Calling document API's in the connectedCallback **NEVER** in the constructor!!!
- Proper bindings

### Testing Instructions
Apply patch, change browser to Code mirror and check if everything still works ok


### Expected result



### Actual result



### Documentation Changes Required
No, this just moves the code closer to production
